### PR TITLE
Adding JAMF

### PR DESCRIPTION
**Pull request by jinglyjangles**
JAMF logging is the bane of my existence.

### CODE REVIEW

File: data/vendors/jamf.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name = "JAMF"
```
Comment (shellcromancer) [inline data/vendors/jamf.toml:1]: Could you clarify a bit on which Jamf component this relates to if it's not all of them?

File: data/vendors/jamf.toml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+url = "https://www.jamf.com"
+grade = "D"
+issues = [
+    "Can only export logs via the UI, in batches of 500 messages per export.",
+    "If you don't want to use the UI, you have to manually poll each machine in your environment to get the machines audit logs.",
+    "JAMF Compliance Reporter has better audit logs. (Not part of the base JAMF install)",
+    "Admin audit log events extremely sparse for the events that have them.",
+]
+gating_mechanism = "JAMF Compliance Reporter & JAMF PRO"
```
Comment (shellcromancer) [inline data/vendors/jamf.toml:10]: Let's reword this as `Higher Tier (Enterprise+) & Additional SKUs (Compliance Reporter)`

File: data/vendors/jamf.toml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+source = "https://learn.jamf.com/en-US/bundle/jamf-protect-documentation/page/Audit_Logs.html"
```
Comment (shellcromancer) [inline data/vendors/jamf.toml:11]: Can you leave a few other sources here as comments? I'm working on adding >1 source links so it'll make it easier for that conversion down the line 😄 

- https://developer.jamf.com/developer-guide/docs/jamf-log-stream
- https://developer.jamf.com/developer-guide/docs/webhooks

**Review by shellcromancer — COMMENTED** [review]
Thanks for submitting the PR! Left a few comments on the wording of the changes in there - additionally there are some ways device check-in, and other specific event information  by configuring the [Jamf Webhooks](https://developer.jamf.com/developer-guide/docs/webhooks) - available for Jamf Pro customers - to go to your systems. Considering that it might warrant a higher grade in the D+ to C- range?

**Comment by jinglyjangles** [comment]
My main complaint with JAMF Pro/etc is that they don't actually provide a Admin audit log. Yes I can pull what each machine did, but I can't pull or log what a admin did in the UI. Which is my main reason for giving it a D.

**Review by shellcromancer — APPROVED** [review]
